src/cli.rs

```diff
@@ -1,17 +1,65 @@
 use bfint::interpret;
-use std::io::{self, Read};
+use std::fs;
+use std::io::{self};
 
-fn get_code_from_stdin() -> io::Result<Vec<u8>> {
-    let mut buf: Vec<u8> = vec![];
-    io::stdin().read_to_end(&mut buf).expect("aboba");
-    Ok(buf)
+use clap::{Args, Parser, Subcommand};
+
+#[derive(Parser)]
+#[command(author, version, about, long_about = None)]
+#[command(propagate_version = true)]
+struct Cli {
+    #[command(subcommand)]
+    command: Commands,
+}
+
+#[derive(Subcommand)]
+enum Commands {
```

Comment:
subcommand is really needed ?
there is only one...

Reply:
Okay, then u wanna see next interface?:
```
bfint-cli -c "some code here"
bfint-cli ./filepath_to_bf_code.b
```

Reply:
yes!!11!